sdk/core/azure-core-amqp/inc/azure/core/amqp/connection.hpp

```diff
@@ -159,12 +184,14 @@ namespace Azure { namespace Core { namespace Amqp { namespace _internal {
      * @remarks Note that this function should only be overriden if the application is listening
      * on the connection.
      */
+    // LCOV_EXCL_START
```

Comment:
We ignore code coverage pretty often, is here really no way covering these?
I understand if that is around 3rd party library failing, and we don't know how to trigger the failure, but here, is it not possible to cover?

Reply:
This default exists so a customer creating connection events doesn't need to provide an OnNewEndpoint callback. But there's no way of triggering that callback to be executed if you aren't implementing a connection listener. 

It may be possible to build a test to force it to be called, but the test wouldn't be meaningful, it would simply exist to ensure that the compiler generated correct code. I'm not sure it's worth the value added.